src/datepicker/styled-components.js

```diff
@@ -40,6 +40,34 @@ export const StyledCalendarContainer = styled(
   },
 );
 
+export const StyledQuickSelectContainer = styled(
+  'div',
+  (props: SharedStylePropsT) => ({
+    maxWidth: '296px',
+    marginLeft: 'auto',
+    marginRight: 'auto',
+    marginBottom: '16px',
```

Comment:
Would it make sense to use theme's `scale600` instead?

Reply:
ah right - i need to go through and use theme values. I was using px for prototyping it